code/framework/object/src/main/java/io/cattle/platform/object/meta/impl/DefaultObjectMetaDataManager.java

```diff
@@ -55,14 +56,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.Lists;
+
 public class DefaultObjectMetaDataManager implements ObjectMetaDataManager, SchemaPostProcessor, InitializationTask, Priority {
 
     private static final Logger log = LoggerFactory.getLogger(DefaultObjectMetaDataManager.class);
 
     SchemaFactory schemaFactory;
     List<TypeSet> typeSets;
-    Map<Class<?>,Map<String,Relationship>> relationships = new HashMap<Class<?>, Map<String,Relationship>>();
-    Map<Class<?>,Map<String,Relationship>> relationshipsBothCase = new HashMap<Class<?>, Map<String,Relationship>>();
+    Map<Class<?>,Map<String,List<Relationship>>> relationships = new HashMap<Class<?>, Map<String,List<Relationship>>>();
```

Comment:
Most of the changes in this file are related to refactoring so "relationships" and "relationshipsBothCase" manages a List<Relationship>.  This is due to the host (cluster) <=> host mapping.